lmod_proxy/static/site/js/site.js

```diff
@@ -0,0 +1,129 @@
+$(document).ready(function(){
+
+  // Setup nice click handling for gradebook
+  $("#gradebook").click(function() {
+    if(!$(this).hasClass("selected")) {
+      $(this).select();
+      $(this).addClass("selected");
+    }
+  });
+  $("#gradebook").blur(function() {
+    if($(this).hasClass("selected")) {
+      $(this).removeClass("selected");
+    }
+  });
+
+
+  function post_call(data) {
+    $('#error').hide(100);
+    console.log(data)
+    $.post(location.pathname, data)
+      .done(function(data, status){
+        console.log(data);
+        console.log(status);
+        $('#result-msg').html(data['msg']);
+        var tbl=$("<table/>").attr("id","datatable");
+        var row = "<thead><tr>";
+        if(typeof data['data'][0] !== 'undefined') {
+          $.each(Object.keys(data['data'][0]), function(index, item) {
```

Comment:
that would be a result of unfamiliarity with the `$.each` callback, but don't I have to still specify both to get the value instead of just the property name since it is the second callback parameter, or is there some javascript magic I'm missing?